<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthRegForm</class>
 <widget class="QDialog" name="AuthRegForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>351</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_reg">
      <property name="text">
       <string>Reg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_auth">
      <property name="text">
       <string>Auth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_to_reg">
      <property name="text">
       <string>to Reg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_forgot_password">
      <property name="text">
       <string>Forgot Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>351</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_login">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_password">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_password_check">
      <property name="text">
       <string>Password repeat</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_email">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_password"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_email"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_password_reapet"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_login"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_header">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Window</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_test_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QWidget" name="resetPasswordWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>351</width>
     <height>200</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="resetPasswordLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_verification_code">
      <property name="text">
       <string>Verification Code</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_verification_code"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_new_password">
      <property name="text">
       <string>New Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_new_password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_confirm_new_password">
      <property name="text">
       <string>Confirm New Password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_confirm_new_password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="resetPasswordButtonsLayout">
      <item>
       <widget class="QPushButton" name="pushButton_send_code">
        <property name="text">
         <string>Send Code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset_password">
        <property name="text">
         <string>Reset Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel_reset">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_forgot_password</sender>
   <signal>clicked()</signal>
   <receiver>AuthRegForm</receiver>
   <slot>on_pushButton_forgot_password_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_send_code</sender>
   <signal>clicked()</signal>
   <receiver>AuthRegForm</receiver>
   <slot>on_pushButton_send_code_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset_password</sender>
   <signal>clicked()</signal>
   <receiver>AuthRegForm</receiver>
   <slot>on_pushButton_reset_password_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel_reset</sender>
   <signal>clicked()</signal>
   <receiver>AuthRegForm</receiver>
   <slot>on_pushButton_cancel_reset_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
